packages/core/docs/core/utils/server-side-rendering.md

```diff
@@ -0,0 +1,106 @@
+# Server-side rendering
+
+Sometimes our composables needs to load some data and share that state between client and the server side.
+We provide our custom mechanism to create server-side rendered pages used with composables functions.
+
+## Usage
+
+In the most cases, the composable functions provide SSR support (it is always transparent for developer).
+To build that composable you have to create a new composable-factory with using `useSSR` function.
+
+```js
+import { useSSR } from '@vue-storefront/utils';
+
+export function useExampleFactory(factoryParams) {
```

Comment:
imo factory here is not needed and just adds redundant complexity, just show how to use it with some example use case (using `example` is not the best way to explain stuff as well, show something that can be understood as an idea - simplified product comparable or so)